WeakAurasOptions/RegionOptions/DynamicGroup.lua

```diff
@@ -364,15 +364,17 @@ local function createOptions(id, data)
     borderSpace = {
```

Comment:
Since you touched these options and therefor own them forever, can you implement this TODO for bonus points?

Reply:
I have implemented same options as group region with settings migration.
Also changed how borders are draw to use position of children instead of the borders of the group frame, without this change for example with grow=left, icons start from the middle and cross the border